newfile.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<body>
```

Comment:
this is where you would create a <head>INSIDE</head> tag and inside of that you would put this <link rel="stylesheet" href="./shapes.css"> so it would look like the screen shot that I sent you through slack.  Great job second time around.  Love the ambition.

Reply:
Thanks Carrye!